src/oracles/ChainlinkPriceOracle.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Contributions to cowprotocol repositories for smart contracts must be in `GPL-3.0` license :pray: